tests/test_narrator.py

```diff
@@ -108,7 +108,7 @@ def test_deep_kink(self) -> None:
         mock_adapter = get_mock_adapter()
         narrator = Narrator(adapters=[mock_adapter])
 
-        with narrator.mic_cable_kinked():
+        with narrator.mic_cable_kinked():  # noqa: SIM117
```

Comment:
WUT!?!  you didn't want to make it look terrible?  😁

Reply:
Hahaha i tried it the way it suggested and... yeah. Sometimes i think we know better.